### ISSUE TITLE
Some 68K games run WAY TOO FAST even with 33mhz Quadra 650 ROM?

### ISSUE THREAD

**Issue by that-ben** [mihaip/infinite-mac#198]
Similarly to #42 where HyperCard dissolve transitions played _WAY TOO SLOW_, I'm wondering if something similar could cause some 68K games to run excessively fast (way too fast) under the Basilisk II emulator? e.g. even if you set the speed option to the slowest setting (which would have been completely unplayable on a real 68K Mac) it's still too fast: https://infinitemac.org/run?machine=Quadra%20650&disk=System%207.1&cdrom=https://emulate-in-browser.macintoshrepository.org/d/9421_MyDude.dsk

**Comment by mihaip** [comment]
Basilisk II doesn't have a speed setting (see discussion in [this thread](https://www.emaculation.com/forum/viewtopic.php?t=7191), probably others). It could perhaps be added, but you could also try running the game using the Mac II ROM, which is under Mini vMac, which does support a 1x speed option.

**Comment by that-ben** [comment]
True, though I was wondering if this setting was still a thing in the source code you used? This is the older build of Basilisk II that I still use for floppy disk support and I can slow it down by introducing a tiny delay every couple milliseconds:

![asddasdasdas](https://github.com/mihaip/infinite-mac/assets/13210399/7af89692-4d82-49e8-a093-5d59e701b62c)

**Comment by mihaip** [comment]
Can't seem to find any source code references to that Windows setting. There is a `delay` pref: https://github.com/mihaip/macemu/blob/bca8aab45f5aba9e540d5c5914193fef56dbecd4/BasiliskII/src/prefs_items.cpp#L88

But that was added pretty recently (https://github.com/kanjitalk755/macemu/commit/f07b34a451a8f74373f5d90210842372c5a81b9d) and is not on a code path that is used in the Emscripten build (we enable `USE_CPU_EMUL_SERVICES`)

**Comment by that-ben** [comment]
Ah :ok:

**Comment by mihaip** [comment]
Have you tried [SpeedChopper](https://macintoshgarden.org/apps/speedchopper). I can look into including it on Infinite HD if it helps.

**Comment by that-ben** [comment]
Very good idea, I'll try it.  But I think it should definitely be included in the Infinite HD, given the usefulness and the tiny file size.  Thanks for the suggestion, I had forgotten about those little utilities that helped newer Mac run old games at reasonable speed! It's such an artifact of the past, like decades ago, all games frame rate was based off timers and capped like it's supposed to :P
